metricbeat/module/haproxy/stat/_meta/docs.asciidoc

```diff
@@ -11,8 +12,8 @@ http://www.haproxy.org/download/1.6/doc/management.txt
 
 The following documentation bellow is an extract from the URL above, more specifically from section "9.1. CSV format"
 
-[source]
------------------
+[source,ruby]
```

Comment:
This needs to be fixed. Didn't work without tag. @dedemorton Some suggestion on what the best tag would be to use here?